awscli/customizations/globalargs.py

```diff
@@ -66,6 +58,23 @@ def _resolve_endpoint_url(value):
     return value
 
 
+def resolve_verify_ssl(parsed_args, session, **kwargs):
+    SYSTEM_CA_BUNDLE = '/etc/pki/tls/certs/ca-bundle.crt'
```

Comment:
FYI, when trying to add this in another PR, Guzzle does a decent job of finding a CA bundle on different platforms: https://github.com/guzzle/guzzle/blob/master/src/functions.php#L160-L174